app/presenters/supergroups/service.rb

```diff
@@ -0,0 +1,25 @@
+class Service < Supergroup
```

Comment:
Should this be renamed to `Services` for consistency with the content_purpose_supergroup?

Reply:
There is already a class called Services and rails can't tell the difference between them...

Reply:
Of course there is! Ok.

Reply:
I think we would have to change one of them. Do you know a way around this?

Reply:
The only way I know of is to wrap the superclass and subclasses in a module. So if `Service` became `Supergroup::Services` Rails would be able to find it.